lib/Dialect/Seq/SeqPasses.cpp

```diff
@@ -46,9 +46,6 @@ struct CompRegLower : public OpConversionPattern<CompRegOp> {
     DictionaryAttr regAttrs = reg->getAttrDictionary();
     if (!regAttrs.empty())
       svReg->setAttrs(regAttrs);
```

Comment:
Please manually set the attributes in question, e.g. by building the op with its builder.  Please don't "just pass on" whatever attributes are on the seq register.